README.md

```diff
@@ -84,16 +84,14 @@ If you do not want Terraform to create a new IAM user or you do not have permiss
 
 ```terraform
 module "eks-jx" {
-  source  = "jenkins-x/eks-jx/aws"
-
-  output "jx_requirements" {
-    value = module.eks-jx.jx_requirements
-  }
-
-  vault_user="<your_vault_iam_username>"
+  source     = "jenkins-x/eks-jx/aws"
+  vault_user = "<your_vault_iam_username>"
 }
 ```
 
+You should have your [AWS CLI configured correctly](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+In addition, you should make sure to specify the region via the AWS_REGION environment variable. e.g. `export AWS_REGION=us-east-1`
```

Comment:
I think this is more of a troubleshooting tip, as it does not affect all users. thoughts?

Reply:
I don't know.  I know I got caught on the AWS_REGION thing while having my CLI configured.  So in order to put that in there, to help someone else avoid pain, I figured a little context would also be helpful.

Too much documentation? Is that a thing?  :-P

Reply:
... as in, I have the region configured in my AWS profile but it still failed... so having AWS_REGION is an extra requirement if you don't already have it.

Reply:
something like a warning box, might be better imo (as it may not be required for everyone). It adds value :+1: 
And more documentation is always good, our documentation needs to improve :)

Reply:
But I would like to see what @hferentschik has to say about it.